modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,32 @@
+= Anypoint Monitoring and Visualizer
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Requirements
+
+=== Stateful Set Directories
```

Comment:
These requirements live in the:
```
* xref:install-checklist.adoc[Product Prerequisites]
 ** xref:prereq-platform.adoc[Environment Prerequisites]
 ** xref:prereq-hardware.adoc[*NEEDS UPDATES*: Hardware Prerequisites]
 ** xref:prereq-network.adoc[Network Prerequisites]
```

Reply:
I see the directories and port info are already listed at https://beta.docs.stgx.mulesoft.com/beta-pce/private-cloud/3.0/prereq-hardware , so I've removed them from this file.